recipe/install-rust-std-extra.bat

```diff
@@ -0,0 +1,15 @@
+@echo off
+setlocal
+
+cd rust-std
+
+echo %PKG_NAME% > ./components
+
+bash -lce "install.sh --prefix="%PREFIX%" --destdir="%DESTDIR%""
+
+del "%PREFIX%/lib/rustlib/components"
+del "%PREFIX%/lib/rustlib/install.log"
+del "%PREFIX%/lib/rustlib/rust-installer-version"
+del "%PREFIX%/lib/rustlib/uninstall.sh"
+
+endlocal
```

Comment:
Can you use the contents of `recipe/install-rust-std.bat` here with `s/install-rust-std.sh/install-rust-std-extra.sh/g`